policyengine_us/parameters/gov/states/vt/tax/income/rates/alternative_minimum_tax/income_threshold.yaml

```
@@ -0,0 +1,14 @@
+description: Vermont qualifies filers for the alternative minimum tax with adjusted gross income above this threshold.
```

qualifies implies optionality or that it is a benefit
```suggestion
description: Vermont levies an alternative minimum tax on filers with adjusted gross income above this threshold.
```